engine/runtime/src/main/java/org/enso/interpreter/runtime/data/EnsoFile.java

```diff
@@ -89,6 +90,10 @@ public void delete() throws IOException {
     truffleFile.delete();
   }
 
+  public void move(EnsoFile target, CopyOption... options) throws IOException {
```

Comment:
This would probably benefit from https://github.com/enso-org/enso/pull/3471 which is the final stages of being merged